js_modules/dagit/packages/core/src/app/Permissions.tsx

```diff
@@ -89,22 +89,25 @@ export const extractPermissions = (
 
 export type PermissionsMap = ReturnType<typeof extractPermissions>;
 
-type PermissionsContext = {
+type PermissionsContextType = {
```

Comment:
Also fixing this type, it was colliding with the actual name of the context object.